src/main.js

```diff
@@ -1,4 +1,8 @@
 // Create variables targetting the relevant DOM elements here 👇
+var newCoverImage = document.querySelector('.cover-image');
+var newTitle = document.querySelector("cover-title");
+var newTaglineOne = document.querySelector("tagline-1");
```

Comment:
I like what we did here, can we maybe clean up these names as we move forward and create more variables?